Library/Invoice.cs

```diff
@@ -104,6 +105,16 @@ public byte[] GetPdf(string acceptLanguage = "en-US")
         }
 
         /// <summary>
+        /// Returns a PDF representation of an invoice
+        /// </summary>
+        /// <param name="acceptLanguage">Language for invoice, defaults to en-US.</param>
+        /// <returns></returns>
+        public async Task<byte[]> GetPdfAsync(string acceptLanguage = "en-US")
```

Comment:
I was able to get that revert in. Please let me know if there are any other items I can address. Thanks.